readme.md

```diff
@@ -655,6 +655,15 @@ await got('https://api6.ipify.org', {
 });
 ```
 
+###### lookup
+
+Type: `Function`
+Default `dns.lookup`
+
+Custom lookup function.
```

Comment:
Can we maybe succinctly explain what "lookup" means?

Reply:
Maybe something like: "Custom lookup function. Use it to implement a custom DNS resolution logic"

Reply:
How about:

> Custom DNS resolution logic.